src/main/webapp/upload.html

```diff
@@ -36,7 +39,17 @@ <h1>Upload a Receipt</h1>
       <label id="receipt-filename-label" class="custom-file-label" for="receipt-image-input">Choose file</label>
     </div>
     <small class="form-text text-muted">Only JPEG images under 5 MB are accepted.</small>
-    <br />
+
+    <div id="loading" class="hidden">
+      <div
+        class="ldBar"
```

Comment:
See https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Also could we use "loading" instead of abbreviating?